packages/datadog-instrumentations/src/aws-sdk.js

```diff
@@ -205,3 +206,8 @@ addHook({ name: 'aws-sdk', file: 'lib/core.js', versions: ['>=2.1.35'] }, AWS =>
   shimmer.wrap(AWS.Request.prototype, 'send', wrapRequest)
   return AWS
 })
+
+addHook({ name: '@aws-sdk/client-bedrock-runtime', file: 'dist-cjs/index.js', versions: ['>=3.693.0'] }, AWS => {
+  shimmer.wrap(AWS.Client.prototype, 'send', wrapRequest())
+  return AWS
+})
```

Comment:
we shouldn't need this hook (i think), does the test instrumentation you have work without this?

Reply:
I think it works still but why don't we need it ?

Reply:
all AWS services in the `aws-sdk` or `@aws-sdk/{service}` modules should rely on the `@aws-sdk/smithy-client` or `AWS.Request` functions to make requests to AWS. Since we have those already patched, we don't want to patch the function that's calling the request/client `send` functions

Reply:
also, as an aside - if we were patching here, we would just pass in `wrapRequest` instead of `wrapRequest()`. This is because `shimmer.wrap` takes in a function as its third argument, so it would look like:
```javascript
shimmer.wrap(AWS.Client.prototype, 'send', (originalSend) => wrapRequest(originalSend))
```
However, in JS, if the lambda function you provide just doex `(x) => foo(x)`, i.e. passing a value directly into another function, you can just pass that function in directly:
```javascript
shimmer.wrap(AWS.Client.prototype, 'send', wrapRequest)
```
Calling the function directly, in this case, doesn't pass the original `send` function into it